libs/ui-toolkit/src/components/link/link.tsx

```diff
@@ -16,15 +16,25 @@ export const Link = ({ className, children, ...props }: LinkProps) => {
     'cursor-pointer': props['aria-disabled'] !== true,
     'opacity-50 pointer-events-none': props['aria-disabled'] === true,
   });
+  const shared = {
+    role: 'link',
+    'data-testid': 'link',
+    referrerPolicy: 'strict-origin' as const,
+    className: anchorClassName,
+  };
+
+  // if no href is passed just render a span, this is so that we can wrap an
+  // element with our links styles with a react router link compoment
+  if (!props.href) {
+    return (
+      <span {...shared} {...props}>
+        {children}
+      </span>
+    );
+  }
 
   return (
-    <a
-      role="link"
-      data-testid="link"
```

Comment:
finally. each link should have a different data-testid

Reply:
This is how it was before, so I didn't want to make any changes to data-testid in this PR. Also you can already change the value as  `props` are spread into the compoment after `shared` which will merge them.